src/server.c

```diff
@@ -4655,6 +4654,8 @@ void commandHelpCommand(client *c) {
 "    Return a list of all commands in this Redis server.",
 "INFO <command-name> [<command-name> ...]",
 "    Return details about multiple Redis commands.",
+"DOCS [<command-name> ...]",
+"    Return documentation details about all / multiple Redis commands.",
```

Comment:
The syntax and description should probably follow the same style as the INFO sub-command above.

Reply:
but INFO has a mandatory argument and DOCS doesn't.
this is because the COMMAND command is the only command that has sub-commands but also works as a command on it's own (i hate it).
i.e. if you want the INFO on all commands you just do `COMMAND`, but if you want the DOCS on all command you do COMMAND DOCS.
do you have a better suggestion?

Reply:
Ah, thanks for explaining this.

I think it's fine, but the explanation can be more clear, e.g.

```suggestion
"INFO <command-name> [<command-name> ...]",
"    Return details about multiple Redis commands.",
"DOCS [<command-name> ...]",
"    Return documentation details about multiple Redis commands.",
"    If no command names are given, documentation details for all",
"    commands are returned.",
```

Reply:
Possibly: We could make COMMAND INFO without command name args return the same as COMMAND, so it will be symmetrical to COMMAND DOCS. Then later we could deprecate COMMAND without sub-command.

Reply:
ohh, how much i like to deprecate COMMAND (no args).. i wish we could do that... it'll be a big break, and clients will have to use a fallback for old versions..

but you're right. let's upgrade INFO.

Reply:
We may not be able to deprecate COMMAND (no args) but we can describe it as being an alias of COMMAND INFO (no args).